tests/core/filter/editor.js

```diff
@@ -339,6 +339,35 @@
 				assert.isFalse( 'filter' in editor, 'editor.filter was deleted' );
 				assert.isFalse( 'activeFilter' in editor, 'editor.activeFilter was deleted' );
 			} );
+		},
+
+		// (#1330)
+		'test converting margin shortcut to separate styles': function() {
```

Comment:
I think you could go with more unit test like approach, checking only modified method. It is available under `CKEDITOR.filter.transformationTools.splitMarginShorthand`.

Or, there is already a similar test (more like integration one) in filter transformation test: https://github.com/ckeditor/ckeditor4/blob/8fccb33c878b0ec2bcc414d3ca2b1b85c0eafd2e/tests/core/filter/transformations.js#L416

So I'm for the second approach here - introducing similar test to what we have `test margin transformations`, checking all possible cases with `0`s - this are mentioned in #1330. You can also [take a look on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#Syntax).